reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+ARG BASE_CONTAINER=jupyter/pyspark-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Microsoft Recommender Project <RecoDevTeam@service.microsoft.com>"
+
+ARG WORK_PATH=/home/$NB_USER/work
+ARG RECO_PATH=$WORK_PATH/Recommenders
+
+USER root
+
+# Clone repo & discard notebooks that cannot be run in this environment
+RUN git clone --single-branch --branch only_pyspark https://github.com/Microsoft/Recommenders.git && \
```

Comment:
why not set working directory to work_path prior to git clone so you don't need to copy the directory?